pkg/odo/cli/service/create.go

```diff
@@ -281,3 +339,20 @@ func NewCmdServiceCreate(name, fullName string) *cobra.Command {
 	completion.RegisterCommandFlagHandler(serviceCreateCmd, "parameters", completion.ServiceParameterCompletionHandler)
 	return serviceCreateCmd
 }
+
+func gvFromALMExample(example map[string]interface{}) (group, version string) {
+	apiVersion := example["apiVersion"].(string)
+	gv := strings.Split(apiVersion, "/")
```

Comment:
can we confirm the length here to be 2?

Reply:
Sure we can. But can you provide more context about what we should be doing if it's not 2 and why?

Reply:
Maybe use SplitN so more / are not ignored maybe? Or show error if there are more then 2 elements?

Reply:
OK. I was struggling to understand why we are focusing on size being 2. But I understood the point while typing the question. Anyway, look at the comment just before the call to `SplitN` and let me know if it makes sense.